### PR TITLE
Rollout v2 ad serving pipeline to all users - Staging

### PR DESCRIPTION
**Pull request by tmancey**
This feature has been rolled out to 50% of users for several months, we are not rolling out to 100% of users. Thanks. cc @btlechowski

### CODE REVIEW

File: seed/seed.json [hunk 1]
```diff
@@ -512,67 +512,39 @@
...
                 }
             ],
             "filter": {
                 "channel": [
-                    "RELEASE"
+                    "RELEASE",
+                    "NIGHTLY",
+                    "BETA"
                 ],
+				"max_version": "109.1.48.60",
```
Comment (aseren) [inline seed/seed.json:547]: This is the first version with rewards 2.5 enabled.

File: seed/seed.json [hunk 2]
```diff
@@ -644,130 +585,16 @@
...
-                    "AU",
-                    "CA",
-                    "FR",
-                    "DE",
-                    "IE",
-                    "JP",
-                    "NZ",
-                    "GB"
-                ],
+				"min_version": "109.1.48.61",
```
Comment (aseren) [inline seed/seed.json:597]: Is this the first version with rewards 2.5 enabled?

Comment (tmancey) [reply seed/seed.json:597]: This is the first version with Rewards 2.5

**Comment by btlechowski** [comment]
Verification passed on

Brave | 1.52.34 Chromium: 112.0.5615.49 (Official Build) nightly (64-bit) 
-- | --
Revision | bd2a7bcb881c11e8cfe3078709382934e3916914-refs/branch-heads/5615@{#936}
OS | Ubuntu 18.04 LTS

#### Verified ad serving v2 is applied for min version `109.1.48.61`
image

Verified ad serving v2 is actually used for ntp si:
```
[9486:9486:0412/001253.543194:VERBOSE1:eligible_new_tab_page_ads_v2.cc(36)] Get eligible new tab page ads
[9486:9486:0412/001253.600198:VERBOSE1:eligible_new_tab_page_ads_v2.cc(117)] 4 eligible ads out of 4 ads
```

Verified ad serving v2 is actually used for notification ads:
```
[9486:9486:0412/001437.508097:VERBOSE1:eligible_notification_ads_v2.cc(37)] Get eligible notification ads
[9486:9486:0412/001437.546997:VERBOSE1:eligible_notification_ads_v2.cc(117)] 11 eligible ads out of 11 ads

```

Verified ad serving v2 is actually used for inline ads:
```
[9486:9486:0412/001552.323377:VERBOSE1:eligible_inline_content_ads_v2.cc(38)] Get eligible inline content ads
[9486:9486:0412/001552.340336:VERBOSE1:eligible_inline_content_ads_v2.cc(124)] 1 eligible ads out of 1 ads

```

#### Verified ads are disabled for max version `109.1.48.61`

image

Verified ads actually blocked:
```
[12583:12583:0412/002519.109834:VERBOSE2:permission_rule_util.cc(25)] You have exceeded the allowed notification ads per day
[12583:12583:0412/002519.110185:VERBOSE1:notification_ad_serving.cc(101)] Notification ad not served: Not allowed due to permission rules

```

**Pull request merged**